spring-batch-test/src/test/java/org/springframework/batch/test/JobLauncherTestUtilsTests.java

```diff
@@ -17,11 +17,7 @@
 
 import org.junit.Test;
 
-import org.springframework.batch.core.ExitStatus;
-import org.springframework.batch.core.Job;
-import org.springframework.batch.core.JobExecution;
-import org.springframework.batch.core.Step;
-import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.*;
```

Comment:
Can we restore the explicit imports?  Spring's coding standards prefers to avoid `.*` imports.

Reply:
This is fixed